resources/assets/components/Post/__snapshots__/test.js.snap

```diff
@@ -44,9 +44,7 @@ exports[`it renders correctly 1`] = `
           >
             Photo Caption
           </h4>
-          <p>
-            Here is my awesome caption!
-          </p>
+          <p />
```

Comment:
whoa is this a self closing `<p>` tag?

Reply:
oh weird! I just updated the snapshot and it added that in!